src/stactools/met_office_deterministic/stac.py

```diff
@@ -95,8 +182,8 @@ def _create_item_from_assets(
     item = Item(
         id=item_id,
         datetime=datetime.strptime(parsed.group("valid_time"), "%Y%m%dT%H%MZ"),
-        bbox=[-180, -90, 180, 90],  # TODO: calculate extent for UK items
-        geometry=Polygon.from_bounds(-180, -90, 180, 90).model_dump(exclude_none=True),
+        bbox=GLOBAL_BBOX if "global" in collection else UK_BBOX,
+        geometry=GLOBAL_GEOMETRY if "global" in collection else UK_GEOMETRY,
```

Comment:
This can be in a follow-on PR, but there's enough collection-specific functionality that I think it makes sense to make it a `StrEnum`:

```python
class MetOfficeCollection(StrEnum):
    global_surface = "met-office-global-deterministic-10km-surface"
    # etc

    def get_bbox(self) -> list[float]: ...
    def get_geometry(self) -> dict[str, Any]: ...
    def get_variables(self) -> dict[str, dict[str, str]]: ...
```